src/extensions/default/bramble/lib/LinkManagerRemote.js

```diff
@@ -13,8 +16,19 @@
             return true;
         }
 
-        if(!(/\:?\/\//.test(url)) && window._Brackets_LiveDev_Transport) {
+        var pathNav = !(/\:?\/\//.test(url));
+        // `internalNav` handles the special case of fragment ids in the
+        // same html page in preview mode (not tutorial mode)
+        var internalNav = /^#.*/.test(href);
```

Comment:
Can we just use `fragmentId` as the name here vs. `internalNav` if that's what it is?  Also, you should probably allow arbitrary whitespace before the leading `#` and not bother with what comes next (you're just allowing everything...e.g., `/^\s*#/`)